### DISCUSSION TITLE
Can I install Github Copilot on more than one computer?

### DISCUSSION THREAD

**Discussion by ProjectErostribe** [community/community#12157] [Copilot Conversations] [answered]
I have access to Github Copilot and I installed it on my work computer.  I would like to install it on my personal computer.  How do I do so?

**Comment by davecheney** [comment, marked as answer]
You can install the extension on multiple computers. The documentation is here https://github.com/github/copilot-docs

**Reply by dev-zarghami** [reply]
Can I use one account on two devices at the same time?
So that my brother and I can use it together at the moment

**Reply by davecheney** [reply]
GitHub’s terms of service do not permit accounts to be shared.

**Reply by slim-python** [reply]
> GitHub’s terms of service do not permit accounts to be shared.

but can i use it in one device at a time?

**Comment by iamzaidsoomro** [comment]
Yes, you can do so. You will need to log in with your GitHub account after installing the extension in your IDE.
Check out the official documentation for more guidance: https://github.com/github/copilot-docs
